sycl/test-e2e/ESIMD/assert.cpp

```diff
@@ -1,12 +1,10 @@
-// REQUIRES: linux && level_zero
+// TODO: This test is flaky on PVC and stable on gen12. Investigate PVC fail
+// and remove Gen12 limitation.
+// see intel/llvm#11359
+// REQUIRES: linux, level_zero, gpu-intel-gen12
```

Comment:
We don't set `gpu-intel-gen12` in the upstream CI. I'm working on enabling ATS-M run in post-commit, but even that would use `gpu-intel-dg2`.